app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  def index
+    @comments = Comment.all
+  end
+
+  def show
+    @comment = Comment.find(params[:id])
+  end
+  
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.create(comment_params)
+
+    redirect_to comment_path(@comment)
+  end
+
+  def edit
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @comment = Comment.find(params[:id])
+    @comment.update(comment_params)
+
+    redirect_to comment_path(@comment)
+  end
+
+  def destroy
+    @comment = Comment.find(params[:id])
+    @comment.destroy
+
+    redirect_to comments_path
+  end
+
+  # strong params
+  private
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
+end
```

Comment:
You've been able to practice this since completing this assignment but, nevertheless, how would you go about associating a comment with a particular post?